radixdlt-core/radixdlt/src/main/java/org/radix/universe/output/AWSSecretManager.java

```diff
@@ -0,0 +1,176 @@
+package org.radix.universe.output;
+
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.secretsmanager.SecretsManagerClient;
+import software.amazon.awssdk.services.secretsmanager.model.*;
+
+import java.security.Security;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class AWSSecretManager {
+    public static void createSecret(String secretName, String secretValue, String network) {
+        removeBouncyCastleSecurityProvider();
+
+        Region region = Region.EU_WEST_2;
```

Comment:
Make a common constant `Region.EU_WEST_2` and use it in all methods

Reply:
I'd rather see it as a parameter (with a default value of EU_WEST_2, if not specified)